### ISSUE TITLE
Fails to build on 7.10 because of  Ambiguous occurrence ‘traceM’

### ISSUE THREAD

**Issue by Bilalh** [nikita-volkov/neat-interpolation#3]
Fails to build on 7.10,  log

```
src/NeatInterpolation/Prelude.hs:5:5:
    Ambiguous occurrence ‘traceM’
    It could refer to either ‘NeatInterpolation.Prelude.traceM’,
                             defined at src/NeatInterpolation/Prelude.hs:52:1
                          or ‘Exports.traceM’,
                             imported from ‘Debug.Trace’ at src/NeatInterpolation/Prelude.hs:41:1-29

src/NeatInterpolation/Prelude.hs:5:5:
    Conflicting exports for ‘traceM’:
       ‘module Exports’ exports ‘Exports.traceM’
         imported from ‘Debug.Trace’ at src/NeatInterpolation/Prelude.hs:41:1-29
       ‘traceM’ exports ‘NeatInterpolation.Prelude.traceM’
         defined at src/NeatInterpolation/Prelude.hs:52:1
```

**Comment by nikita-volkov** [comment]
Fixed in 0.2.2.1

**Comment by Bilalh** [comment]
thanks
